### ISSUE TITLE
New rdkit smiles notation for metal-centre stereochemistry could be useful for building block preparation

### ISSUE THREAD

**Issue by andrewtarzia** [lukasturcani/stk#533]
The discussion here https://www.rdkit.org/docs/RDKit_Book.html#support-for-non-tetrahedral-atomic-stereochemistry is what I mean in the title. It is possible that we could allow this in the documentation to build building blocks.

**Comment by lukasturcani** [comment]
wont this work by default? rdkit handles all smiles stuff for us. is there anything we need to do on our end?

**Comment by andrewtarzia** [comment]
Yes, I believe so - it was more to just provide a clear example of this and perhaps alter the Dative bond examples because it might be easier now. I need to look into it.

**Comment by andrewtarzia** [comment]
To make this change, we can remove the new pin (#542) to rdkit version where these changes are introduced, and it requires internal stk changes.
